doc/docs/master/hub/manage-entities/manage-members.md

```diff
@@ -0,0 +1,61 @@
+# Manage Members
```

Comment:
This whole doc can be moved to be part of the orgs doc as members only are relevant in context of orgs.